static/style.css

```diff
@@ -53,3 +53,24 @@ footer.ui.footer {
 .size {
     font-size: 15px;
 }
+
+.ui.input.focus input,
+.ui.input input:focus {
+  border-color: #85B7D9;
+  background: #FFFFFF;
+  color: rgba(0, 0, 0, 0.8);
+  box-shadow: none;
+}
+.ui.input.focus input::-webkit-input-placeholder,
+.ui.input input:focus::-webkit-input-placeholder {
+  color: rgba(128, 128, 128, 1);
+}
+.ui.input.focus input::-moz-placeholder,
```

Comment:
probably, I just copy and pasted all that from the source, wasn't sure what it does